### ISSUE TITLE
1.22 updates Twig to 3.4.3

### ISSUE THREAD

**Issue by Levdbas** [timber/timber#2672]
## Expected behavior
Updating to 1.22 should install twig to 2.15.3

## Actual behavior
Updating to 1.22 updates twig to 3.4.3

This introduces breaking changes like a missing `filter` tag

First reported: https://github.com/timber/timber/pull/2658#issuecomment-1289531520

See https://getcomposer.org/doc/articles/versions.md#summary.

So this
```json
"twig/twig": ">=1.44.7 || ^2.10",
````

![afbeelding](https://user-images.githubusercontent.com/17764157/200786504-2273d56c-9af3-4643-9c62-2a172ac6c5f9.png)

Leads to the following packages: https://semver.madewithlove.com/?package=twig%2Ftwig&constraint=%3E%3D1.44.7+||+^2.10&stability=stable

## Steps to reproduce behavior

```bash
composer create-project roots/bedrock .
composer require timber/timber
```

## What version of WordPress, PHP and Timber are you using?

WordPress 6.1
PHP 8.0.22
Timber 1.22

## How did you install Timber? (for example, from GitHub, Composer/Packagist, WP.org?)
Composer

**Comment by xavivars** [comment]
Same thing is happening. 

`"twig/twig": ">=1.44.7 || ^2.10",`
should be
`"twig/twig": "^1.44.7 || ^2.10",`

otherwise, twig 3 is installed (as it's > than 1.44.7)

**Comment by gchtr** [comment]
Thanks for your insights and your help! And thanks for the link to the Semver checker, @Levdbas, that’s really helpful.

This will be fixed in #2679. I’ll see that we can publish a hotfix release for this as soon as possible.

**Comment by jarednova** [comment]
Yep, in the midst of a deploy right now! Thanks to @Levdbas @xavivars and @gchtr for serving this all up on a silver platter for me
